generate.sh

```diff
@@ -106,7 +109,6 @@ fi
   "--noshow_progress" \
   "--noshow_loading_progress" \
   "--output_groups=${OUTPUT_GROUPS}" \
-  "$@" \
```

Comment:
I have no idea why this line existed. Please let me know if there's a reason to keep.

Reply:
This line is for supplying additional bazel flags. I have reinstated it now.